File: sum_test.py
import unittest

def sum(a, b):
  """Returns the sum of two numbers."""
  return a + b

class TestSum(unittest.TestCase):
  def test_sum_positive_numbers(self):
    result = sum(2, 3)
    self.assertEqual(result, 5)

  def test_sum_negative_numbers(self):
    result = sum(-2, -3)
    self.assertEqual(result, -5)

if __name__ == "__main__":
  unittest.main()

File: sum_test.go
package sum

import (
	"testing"
)

func sum(a int, b int) int {
	return a + b
}
func TestSum(t *testing.T) {
	result := sum(2, 3)
	expected := 5

	if result != expected {
		t.Errorf("Sum(2, 3) expected %d, got %d", expected, result)
	}
}
